backend/typescript/models/person.pgmodel.ts

```diff
@@ -0,0 +1,10 @@
+import { Column, Model, Table } from "sequelize-typescript";
```

Comment:
Had issues using `sequelize` directly with TypeScript so went with this [package](sequelize-typescript). I recommend reading the docs.

Reply:
Link doesn't seem to work

Reply:
Sorry about that! Updated now, please check again

Reply:
Works now